### PR TITLE
Feat/login signup

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
Designed landing page of E-com for Plants and Planters
Designed log in and sign up page

### CODE REVIEW

File: Components/Navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  width: 1.3rem;
+  height: 1.3rem;
+  font-size: 1.1rem;
+}
+.search-bar{
+  display: flex;
+  align-items: center;
+  flex-grow: 1;
+  position: relative;
+  background-color:  #f7f7fd;
```
Comment (haarsh24) [inline Components/Navbar/navbar.css:67]: Declare background color also

File: Components/Navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+}
+/* .sidebar-wrapper,
+.sidebar {
+  height: 100vh;
+    position: fixed;
+    top: 40px;
+    left: 0;
+} */
+.sidebar {
+  /* max-width: 300px;
```
Comment (haarsh24) [inline Components/Navbar/navbar.css:134]: If they are not useful , you can remove commented code

**Review by haarsh24 — APPROVED** [review]
Some minor changes. 
Rest all looks good.

File: Components/Navbar/navbar.css (hunk 2 above)
Comment (SHIKHASINGH1506) [reply Components/Navbar/navbar.css:134]: These styles are required for toggle button on landing page. Yet to be developed

File: Components/Login/login.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
+.login-wrapper {
+  height: calc(100vh - 63px);
+}
+.login-modal {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  /* background-color: #F7F7F7; */
```
Comment (arjunbharti) [inline Components/Login/login.css:8]: Remove unwanted comments

File: Components/Login/login.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+              id="sidebar-toggler"
+              viewBox="0 0 100 40"
+              width="40"
+              height="40"
+            >
+              <rect width="60" height="10"></rect>
+              <rect y="20" width="60" height="10"></rect>
+              <rect y="40" width="60" height="10"></rect>
+            </svg>
+            <!-- <button class="sidebar-toggler" id="sidebar-toggler">Up</button> -->
```
Comment (arjunbharti) [inline Components/Login/login.html]: Here too

File: Components/Navbar/navbar.css [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+  opacity: 0;
+  transition: all 0.3s cubic-bezier(0.25, 0.46, 0.43, 0.96);
+}
+/* .sidebar-wrapper,
+.sidebar {
+  height: 100vh;
+    position: fixed;
+    top: 40px;
+    left: 0;
+} */
```
Comment (arjunbharti) [inline Components/Navbar/navbar.css:132]: Unwanted comment

**Review by arjunbharti — COMMENTED** [review]
UI is neat and clean. Good looks good, just need some minor changes.

**Comment by SHIKHASINGH1506** [comment]
Fixed the reviewed comments and merged

**Pull request merged**